src/main/resources/assets/bashfiremakerock/lang/de_de.lang

```diff
@@ -0,0 +1,9 @@
+config.bashfiremakerock:title=Hauptmenü
```

Comment:
It's not really a main menu, it's the main category of the config file.

Reply:
Changed it to "Allgemein"